GOVERNANCE.md

```diff
@@ -220,15 +221,15 @@ are made according to the following rules:
   decision-making process outlined above, though "reasonable time" is likely extended
   to at least a week.
 
-- **Changes to the specification** require a dedicated issue on our
-  [issue tracker](https://github.com/zarr-developers/zarr-specs/issues) and even more
-  time than an API change. The appropriate length of time is currently under discussion.
+- **Changes to the specification** follow the [ZEP](https://zarr.dev/zeps)
+  process. Please read [ZEP0000](https://zarr.dev/zeps/active/ZEP0000.html)
+  for details and instructions to submit a new ZEP.
 
 - **Changes to this governance model or our mission, vision, and values**
   require a  dedicated issue on our [issue tracker](https://github.com/zarr-developers/governance/issues)
-  and follow the decision-making process outlined above, with "reasonable time" being
-  at least two weeks. However, if there is unanimous agreement from core developers on the change,
-  it can move forward faster.
+  and follow the [ZEP](https://zarr.dev/zeps) process, with "reasonable time"
```

Comment:
Changes to governance follow ZEP? But then doesn't this need a ZEP?

Reply:
It could.
However, since we've been using the ZEP process for more than a year now, I think we can _mention_ the ZEP process in governance without filing a new ZEP.

The subsequent changes to the governance document should require a new ZEP.